package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@optimizely/react-sdk",
   "version": "2.9.1",
-  "description": "React SDK for Optimizely Full Stack and Optimizely Rollouts",
+  "description": "React SDK for Optimizely Optimizely Feature Experimentation and Optimizely Rollouts",
```

Comment:
I think this might be:  
React SDK for Optimizely Feature Experimentation, Optimizely Full Stack (legacy), and Optimizely Rollouts

Would that be okay? 

cc @russell-loube-optimizely @griffincox-optimizely

Reply:
Yep. See my suggestion below.